Tests/test_Tutorial.py

```diff
@@ -258,6 +259,20 @@ def test_doctests(self):
     def tearDown(self):
         global original_path
         os.chdir(original_path)
+        # files currently don't get created during test with python3.5 and pypy
```

Comment:
Right now you assume the unwanted files come in batches, and only test the existence of one file from each group. Would it be cleaner to build a list of unwanted potential files, and then loop over the list deleting any present?